elm-package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "3.1.1",
+    "version": "4.0.0",
```

Comment:
💭 The last major version bump to 3.x.x prompted a short discussion on Slack: https://noredink.slack.com/archives/C0VVDLEES/p1520961722000597

The suggestion there was to reserve future major version bumps for language changes, or removal of widget versions that are no longer used anywhere in the NRI codebase. So I think this would be a good idea:

> Also this change led to an API change. It looked like we were ignoring a class that we were setting via a bool. i thought it made sense to remove the unused bool.

Also, if we feel the major versions policy above doesn't make sense we should change it, and if we do feel it makes sense we should document it. I'm happy to own either depending on your feeling on it with regards to this PR.

Reply:
Interesting!  🤔 it seems a little weird to limit versioning such that:

- we keep the unused bool in V1 & release without the bool as V2
- we swap all the monolith/content creation uses over to V2
- we kill the V1 version and do a major release

I think the major release at removal looks like work that won't ever happen/will be a chore. Like, after I update upgrade the monolith's dependency on this package, is it then on me to revisit this library and see if I can remove V1, even though I have no idea what content creation's needs are? I guess that's more of a question of general long term package management plans... heyo internal! 😄

For now, I'll add the bool back in and get that version number back to a minor change. 

Thanks!

Reply:
Done with the Checkbox.v2 change!

What do you think about Nri.Colors.Extra not having module-level versioning? should it?

Reply:
Some context you might already have: I believe the primary concern behind having per-widget versioning and disallowing breaking changes is preventing this scenario:

1. I make a breaking change of noredink-ui. 
2. You'd like to make a minor change to some widget in noredink-ui you need for a story you're working on in the monolith. You're now blocked from using this change until the monolith has been changed to deal with the unrelated breaking changes introduced in (1). Ideally I already created a PR for that, but that PR might still take a while getting merged.

Granted, for this PR that doesn't seem a huge concern, as the 'breaking' change is super straightforward.

Good question with regards to who's responsible for taking old versions out of noredink-ui again. Perhaps we simply shouldn't care about those old versions sticking around. After all we also don't care about old versions of libraries sticking around on `elm package` or `npm`. Technically those repositories would be leaner if versions of packages no-one uses anymore get removed, but nobody seems to  think setting that up is worth the effort.

> What do you think about Nri.Colors.Extra not having module-level versioning? should it?

I think it's fine the way it is now. If we ever get to the point we want to make a breaking change in that module, we can decide at that point whether to start versioning the module after all or make a breaking release.

Reply:
I'm very happy to not care about old versions! Maybe we can remove old versions when there's a new elm version or only if we are working on managing dependencies?

Thanks! Will keep `Nri.Ui.Colors.Extra` as is.